### PR TITLE
Compile for ES5/CommonJS support

### PR DESCRIPTION
**Pull request by beck**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by alexpattison** [comment]
This looks solid

File: package.json [hunk 1]
```diff
@@ -2,10 +2,13 @@
...
   "version": "0.0.5",
   "description": "A module extending the Fetch API with some useful default error handling and hooks.",
-  "main": "src/stipulate.js",
+  "main": "cjs/stipulate.js",
+  "module": "src/stipulate.js",
   "scripts": {
-    "test": "babel-node node_modules/.bin/isparta cover --root src/ --report text --report lcov --report html node_modules/.bin/_mocha --",
-    "coveralls": "cat coverage/lcov.info | node_modules/.bin/coveralls"
+    "compile-cjs": "scripts/compile-common-js.sh",
+    "coveralls": "cat coverage/lcov.info | node_modules/.bin/coveralls",
```
Comment (stefanor) [inline package.json:9]: UUOC (unnecessary use of cat)

**Review by stefanor — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (beck) [reply package.json:9]: IMO most of the scripts here can use improvement. That test script made my eye twitch.

**Pull request merged**